### DISCUSSION TITLE
Changing Config file

### DISCUSSION THREAD

**Discussion by KennardTV** [kubilus1/autoortho#104] [Q&A]
Is there anyway of changing the ortho cfg file from my C: drive? I have it set to my F: Drive but it still goes to my C: drive with about 20gb left if I try to install NA its 24gb. So can I config the files to download somewhere else?

**Comment by sergiolamares** [comment]
I've pointed Autoortho folders to another SSD drive, in the X-Plane Scnery config, and created symbolic links on X-Plane's Custom Scenery folder...

**Reply by BradleyGother** [reply]
Is there any chance to make a more advanced step by step instruction manual on this subject for us please. screenshots & a text file will be much appreciated. Or a can post a how to video on YT?
